UKHO.PKSWireMockService.API/IWireMockService.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace UKHO.PKSWireMock.API
+{
+    public interface IWireMockService
+    {
+        void Start();
+        void Stop();
+    }
+}
+
+
```

Comment:
inconsistent spacing 
```suggestion
﻿namespace UKHO.PKSWireMock.API
{
    public interface IWireMockService
    {
        void Start();

        void Stop();
    }
}

```

Reply:
Done